macaddress.go

```diff
@@ -0,0 +1,47 @@
+// vim:ts=2:sw=2:et:ai:sts=2
+package main
+
+import (
+	"errors"
+	"fmt"
+	"net"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+type macAddressValue struct{ v *net.HardwareAddr }
+
+func newMacAddressValue(p *net.HardwareAddr) *macAddressValue {
+	return &macAddressValue{p}
+}
+
+func (f *macAddressValue) Set(s string) error {
+	if s == "broadcast" || s == "all" {
+		s = "ff:ff:ff:ff:ff:ff"
+	}
+	if s == "local" {
+		s = "00:b0:52:00:00:01"
+	}
```

Comment:
I would consider making these special addresses net.HardwareAddr pseudo-constants, e.g.
```go
var (
	bcastMAC = net.HardwareAddr{0xff, 0xff, 0xff, 0xff, 0xff, 0xff}
	localMAC = net.HardwareAddr{0x00, 0xb0, 0x52, 0x00, 0x00, 0x01}
)
```
... and only call `net.ParseMAC` if not one of the predefined aliases.

Reply:
done